storage/namespace.go

```diff
@@ -189,8 +189,6 @@ func newDatabaseNamespace(
 
 	iops := opts.InstrumentOptions()
 	logger := iops.Logger().WithFields(xlog.NewLogField("namespace", id.String()))
-	iops = iops.SetLogger(logger)
-	opts = opts.SetInstrumentOptions(iops)
```

Comment:
No longer need to remove this now that setting storage options does not recreate a series pool no?

Reply:
Yep, will re-add.

Reply:
done